crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -563,7 +563,8 @@ impl Entities {
         self.len = 0;
     }
 
-    /// Returns `Ok(Location { archetype: Archetype::invalid(), index: undefined })` for pending entities.
+    /// Returns the location of an [`Entity`].
+    /// Note: for pending entities, returns `Some(EntityLocation::INVALID)`.
```

Comment:
This could probably be more specific about what a pending entity is.

Reply:
I think struct's doc comment is a better place to document this.